MIPs/mip-revokePermissions.md

```diff
@@ -0,0 +1,108 @@
+---
+MIP: 1
```

Comment:
Just in case: A reminder to increment the MIP number once this PR is ready.

cc @adonesky1

Reply:
The file and MIP number should be left as x and will be changed when merged [per the contributing file](https://github.com/MetaMask/metamask-improvement-proposals/blob/main/CONTRIBUTING.md#step-4-add-your-proposal-to-your-branch)

Reply:
Ok thanks, will update

Reply:
Just pushed the updates